sampleChangerApp/src/converter.cpp

```diff
@@ -68,7 +78,7 @@ void converter::loadRackDefs(const char* fname)
 {
     TiXmlDocument doc(fname);
     if (!doc.LoadFile()) {
-        errlogPrintf("sampleChanger: Unable to open rack defs file \"%s\": %s\n", fname, doc.ErrorDesc());
+        printError("sampleChanger: Unable to open rack defs file \"%s\": %s\n", fname, doc.ErrorDesc());
```

Comment:
Can we get row number as well here? `doc.ErrorRow` should be able to fetch row number.